src/video_core/command_classes/sync_manager.cpp

```diff
@@ -38,7 +38,7 @@ u32 SyncptIncrManager::IncrementWhenDone(u32 class_id, u32 id) {
 }
 
 void SyncptIncrManager::SignalDone(u32 handle) {
-    const auto done_incr =
+    const auto& done_incr =
```

Comment:
```suggestion
    const auto done_incr =
```

find_if always returns the iterator by value, so this is misleading